rest/route/project_copy.go

```
@@ -0,0 +1,81 @@
+package route
+
+import (
+	"context"
+	"net/http"
+
+	"github.com/evergreen-ci/evergreen/rest/data"
+	"github.com/evergreen-ci/evergreen/rest/model"
+	"github.com/evergreen-ci/gimlet"
+	"github.com/pkg/errors"
+)
+
+////////////////////////////////////////////////////////////////////////
+//
+// POST /rest/v2/projects/{project_id}/copy
+
+type projectCopyHandler struct {
+	oldProjectId string
+	newProjectId string
+	sc           data.Connector
+}
+
+func makeCopyProject(sc data.Connector) gimlet.RouteHandler {
+	return &projectCopyHandler{
+		sc: sc,
+	}
+}
+
+func (p *projectCopyHandler) Factory() gimlet.RouteHandler {
+	return &projectCopyHandler{
+		sc: p.sc,
+	}
+}
+
+func (p *projectCopyHandler) Parse(ctx context.Context, r *http.Request) error {
+	p.oldProjectId = gimlet.GetVars(r)["project_id"]
+	p.newProjectId = r.FormValue("new_project")
```

should we return an error here if new_project isn't specified?